### DISCUSSION TITLE
EB Cell center vs Cell Centroid and Face Center vs Face Centroid

### DISCUSSION THREAD

**Discussion by ankithadas** [AMReX-Codes/amrex#3720] [Q&A] [answered]
Dear AMReX team,

What is the difference between EB cell center and cell centroid ? For example, `MLEBABecLap` use `Location` enum to specify the location of phi (cell center or cell centroid) and beta (face center or face centroid). In this context, what is the difference between center and centroid?

Any answers would be much appreciated! 

Kind regards,
Ankith

**Comment by WeiqunZhang** [comment, marked as answer]
Centroid is the integral of coordinate over the non-covered region. If a cell is from -0.5 to 0.5, and the left half of the cell is covered, the centroid is at 0.25.

Center is just the center as if there is no EB. Or it can be defined as the integral of coordinate over the whole cell including both covered and non-covered. Since the cells we have are rectangular, the integral is of course trivial.
